client/src/pages/Profile/functions.jsx

```diff
@@ -158,15 +201,21 @@ export function capitalizeFirstLetter(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
-export function getFroshData() {
-  return {
-    froshGroupIcon: 'λ',
-    froshGroup: 'Lambda',
-    firstName: 'James',
-    lastName: 'Kokoska',
-    discipline: 'Computer Engineering',
-    email: 'test.email@mail.utoronto.com',
-  };
+export async function getFroshData() {
+  try {
+    const response = await axios.get('/user/info');
+    const splitName = response.data.user.name.trim().split(/\s+/);
+    return {
+      froshGroupIcon: 'λ',
+      froshGroup: response.data.user.froshGroup,
+      firstName: splitName[0],
+      lastName: splitName[1],
+      discipline: response.data.user.discipline,
+      email: response.data.user.email,
```

Comment:
froshGroupIcon currently not defined in froshGroupModel schema. Once the schema is fixed, the line will need to be changed to reference the API response.